app/uk/gov/hmrc/gform/graph/processor/UserCtxEvaluatorProcessor.scala

```diff
@@ -32,8 +32,25 @@ object UserCtxEvaluatorProcessor extends IdentifierExtractor {
     (retrievals, userField) match {
       case (AuthenticatedRetrievals(_, enrolments, _, _, _), UserField.EnrolledIdentifier) =>
         authorizedEnrolmentValue(enrolments, authConfig)
-      case (AuthenticatedRetrievals(_, enrolments, _, _, _), UserField.Enrolment(sn, in)) =>
-        extractIdentifier(enrolments, sn, in)
```

Comment:
No one is using `extractIdentifier` now, so it should be removed

Reply:
Removed, thanks